hugegraph-core/src/main/java/com/baidu/hugegraph/analyzer/JcsegAnalyzer.java

```diff
@@ -45,29 +44,35 @@ public class JcsegAnalyzer implements Analyzer {
             "Complex"
     );
 
-    private static final JcsegTaskConfig CONFIG = new JcsegTaskConfig();
+    private static final SegmenterConfig CONFIG = new SegmenterConfig();
     private static final ADictionary DIC =
-            DictionaryFactory.createDefaultDictionary(new JcsegTaskConfig());
+        DictionaryFactory.createDefaultDictionary(CONFIG);
```

Comment:
If this line does not exceed 100 letters, consider merging into one line (ignore the comment  if > 100)